### ISSUE TITLE
POD restarts when applying wasm deploy

### ISSUE THREAD

**Issue by sriramcms** [solo-io/wasme#104]
Hi:

I went through the sample tutorial to add a wasm filter to the bookinfo sample in istio service mesh 1.5.1. 
https://docs.solo.io/web-assembly-hub/latest/tutorial_code/

Does the wasm deploy force a pod restart while applying the filter.  Is this observation correct ?  I see the old pods terminating and new ones getting created -- for example if i am applying this on the details pod in the bookinfo example.

**Comment by yuval-k** [comment]
it is for now as we need to apply some annotations to the pod.
once https://github.com/solo-io/wasme/pull/95 is merged, this will no longer be the case.

**Comment by sriramcms** [comment]
Super. Thanks for the prompt response. Will wait for the new release.

**Comment by sriramcms** [comment]
Just a FYI on what i observed --   istio 1.5.1 , minikube and kubernetes 1.17.3 

A few more observation on constant POD restarts due to constantly applying newer versions of the wasm filters  

1. The kubernetes API server stops responding and to fix it I end up doing a minikube stop and start to get a working cluster again
2.  If for example I am applying this on the petstore app 

First time when i apply wasme deploy to petstore  - hangs and does not succeed 
*****************************************************************************************
INFO[0003] added image to cache config...                cache="{wasme-cache wasme}" image="webassemblyhub.io/sriramcm/demo-add-header:v0.7"
INFO[0003] waiting for event with timeout 1m0s
WARN[0004] event err: expected 1 image-ready events for image webassemblyhub.io/sriramcm/demo-add-header:v0.7, only found 0
WARN[0005] event err: expected 1 image-ready events for image webassemblyhub.io/sriramcm/demo-add-header:v0.7, only found 0

petstore istio - proxy logs looks like below
***********************************************
s Pilot running?): cds updates: 1 successful, 0 rejected; lds updates: 0 successful, 1 rejected
2020-04-18T19:11:02.222499Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 1 successful, 0 rejected; lds updates: 0 successful, 1 rejected
2020-04-18T19:11:04.223060Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 1 successful, 0 rejected; lds updates: 0 successful, 1 rejected
2020-04-18T19:11:06.222974Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 1 

i need to kill it and then run wasme deploy again to suceed 

Also a few wasme undeploys and deploys to get a successful POD with filter -- 

Basically i found sometimes it was erratic.

**Comment by Sodman** [comment]
Related to https://github.com/solo-io/wasm/issues/67
